android/KMEA/app/src/main/java/com/keyman/engine/KMManager.java

```diff
@@ -257,10 +257,13 @@ public int toInt() {
   // haptic feedback disabled for hardware keystrokes
   private static boolean mayHaveHapticFeedback = false;
 
-  // Special override for when keyboard is entering a password text field.
-  // When mayPredictOverride is true, the option {'mayPredict' = false} is set in the lm-layer
+  // Special overrides to temporarily disable predictions when editing a password field.
```

Comment:
maybe call these variables or flag instead of overrides (just because overrides sounds like Java method override)